.github/workflows/ci.yml

```diff
@@ -29,29 +29,30 @@ jobs:
         with:
           key: ${{ github.job }}-ubuntu-latest
           variant: sccache
+      - uses: seanmiddleditch/gha-setup-ninja@16b940825621068d98711680b6c3ff92201f8fc0 # v3
       - uses: lukka/run-cmake@c2b72aff009141774c5a5fabe74ea46c8c04d9c4 # v10.6
         with:
-          configurePreset: "ContinuousIntegration"
-          buildPreset: "ContinuousIntegrationWithPackage"
-          testPreset: "ContinuousIntegration"
+          configurePreset: "host"
+          buildPreset: "host-Debug-WithPackage"
+          testPreset: "host"
           configurePresetAdditionalArgs: "['-DCMAKE_C_COMPILER_LAUNCHER=sccache', '-DCMAKE_CXX_COMPILER_LAUNCHER=sccache']"
       - uses: actions/upload-artifact@0b7f8abb1508181956e8e162db84b466c27e18ce # v3.1.2
         with:
           name: emil
-          path: build/ContinuousIntegration/emil-*-Linux.tar.gz
+          path: build/host/emil-*-Linux.tar.gz
           if-no-files-found: error
       - name: Upload test logs
         if: ${{ failure() }}
         uses: actions/upload-artifact@0b7f8abb1508181956e8e162db84b466c27e18ce # v3.1.2
         with:
           name: test-logs
-          path: build/ContinuousIntegration/Testing/Temporary/
-  host_build_test:
+          path: build/host/Testing/Temporary/
+  host_build_test_windows:
     name: Host Build & Test
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        os: [macos-latest, windows-latest, windows-2019]
+        os: [windows-latest, windows-2019, macos-latest]
```

Comment:
Wherever no sane sorting can be applied, I always keep items sorted alphabetically.

Reply:
There's a whole history of why this was touched, torn apart, put back again, but now I'll re-apply alphabetic order